File: Resturaunt/ResturantAPI/ResturantAPI/Models/Score.cs
using System.ComponentModel.DataAnnotations;

namespace ResturantAPI.Models;

public class Score
{
    [Key]
    public int? Id { get; set; }
    public int? score { get; set; }
    public DateTime? date_submit { get; set; }
    public int? RestId { get; set; }
}

File: RestFront/src/app/models/score.ts
export class Score {
    id: number
    score: number
    restId: number
    date_submit: Date

    constructor (
        id: number,
        score: number,
        date_submit: Date,
        restId: number
    ) {
        this.id = id;
        this.score = score;
        this.date_submit = date_submit;
        this.restId = restId;
    }
}